### ISSUE TITLE
Inquiry Regarding Image Transforms

### ISSUE THREAD

**Issue by SpaceQuark** [prov-gigapath/prov-gigapath#74]
Hello! Thank you so much for sharing this amazing project! I was curious the image transformation pipeline you provided:

    transform = transforms.Compose(
    [
        transforms.Resize(256, interpolation=transforms.InterpolationMode.BICUBIC),
        transforms.CenterCrop(224),
        transforms.ToTensor(),
        transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
    ])
    
It appears that some image details may be lost during the resizing and center cropping steps. Could you please explain why you opted to resize the images to 256x256 instead of directly resizing to 224x224? I am particularly interested in understanding if this decision was made to address any potential issues or benefits that may not be immediately obvious.

**Comment by bhosalems** [comment]
Yes, and wasn't the model trained on 256x256, I don't understand why the model size is 224x224.
